boards/beetle32-c3.json

```
@@ -0,0 +1,39 @@
+{
+    "build": {
+      "arduino":{
+        "ldscript": "esp32c3_out.ld"
+      },
+      "core": "esp32",
+      "f_cpu": "160000000L",
+      "f_flash": "80000000L",
+      "flash_mode": "qio",
+      "extra_flags": [
+        "-DARDUINO_ESP32C3_DEV",
+        "-DARDUINO_USB_CDC_ON_BOOT=1"
```

In framework-arduinoespressif32\platform.txt the ARDUINO_USB_MODE is set. Without it i had no "Serial".
```suggestion
        "-DARDUINO_ESP32C3_DEV",
        "-DARDUINO_USB_MODE=1",
        "-DARDUINO_USB_CDC_ON_BOOT=1"
```